CHANGELOG.md

```diff
@@ -1,12 +1,17 @@
 # Changelog
 
-# 33.0.0
+# 34.0.0
```

Comment:
```suggestion
## 34.0.0
```

Reply:
Ah - made the same mistake I fixed. Sorry! 😅